sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/ConsumerSpEL.java

```diff
@@ -57,4 +74,31 @@ public void evaluateAssign(Consumer consumer, Collection<TopicPartition> topicPa
     mapContext.setVariable("tp", topicPartitions);
     assignExpression.getValue(mapContext);
   }
+
+  private boolean hasTimestamp(){
```

Comment:
This function is used to check whether `ConsumerRecord.class` has a method `timestamp` which returns `Long.TYPE`. Kafka0.9 doesn't have it, Kafka0.10 has. The check should be complete as
```
hasEventTimestamp = timestampMethod != null
          && timestampMethod.getReturnType().equals(Long.TYPE);
```
Right?